CMake/version.cmake

```diff
@@ -67,8 +67,14 @@ configure_file(
   ${CMAKE_CURRENT_SOURCE_DIR}/CMake/config/version.json.in
   ${XDNA_VERSION_JSON_FILE}
 )
-
+if(XDNA_VE2)
+configure_file(
+ ${CMAKE_CURRENT_SOURCE_DIR}/CMake/config/version.h.in
+ ${CMAKE_BINARY_DIR}/gen/version.h
+)
+else()
```

Comment:
What is the use case for generating version.h? We already brought in XRT version in xdna-driver repo. Do you still need to introduce a new way for this?

Reply:
strix xdna uses dkms based mechanism to create version. ve2 driver doesn't use dkms. It gets built with Yocto.
@SravanKumarAllu-xilinx , Don't create version.h file. We can use same json file mechanism to add MODULE_VERSION.
yocto doesn't allow you to create version.json file without releasing, You need to stop creating both version.h & version.json for now.